openlibrary/plugins/importapi/code.py

```diff
@@ -459,7 +459,7 @@ def login(self, authstring):
         if not authstring:
             return
         authstring = authstring.replace("Basic ","")
-        username, password = base64.decodestring(authstring).split(':')
+        username, password = base64.decodebytes(authstring).split(':')
```

Comment:
The project seems to support Python 2 though this code path might not be tested. `decodestring` has to be used in Python 2 and `decodebytes` has to be used in Python 3 to maintain compatibility.

Reply:
Can you suggest what I should I do over here?